p4-16/psa/PSA.mdk

```diff
@@ -1497,6 +1499,65 @@ table.
 A PSA implementation must implement tables that have both a
 `psa_direct_counter` and `psa_direct_meter` property.
 
+A PSA implementation need not support both `psa_implementation` and
+`psa_idle_timeout` properties on the same table.
+
+### Table entry timeout notification {#sec-idle-timeout}
+
+PSA uses the `psa_idle_timeout` to enable a table implementation send
+notifications from the PSA device when a configurable time has passed since an
+entry was last matched. The property may take one of two values --
+`NO_TIMEOUT`, and `NOTIFY_CONTROL`. `NO_TIMEOUT` disables idle timeout support
+for the table and it is the default value when the property is not present.
+`PSA_NOTIFY_CONTROL` enables the notification. A PSA implementation will then
```

Comment:
s/PSA_NOTIFY_CONTROL/NOTIFY_CONTROL/

Reply:
@cc10512 Not sure if this little replacement has been made yet.  I think these changes are ready to commit now.  I will try to look at them with fresh eyes once we are in the 2 week per-release review period, and if there are any minor things like this we can fix them then.

Reply:
I did make it. Refresh?